security/password.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Marc Berhault (marc@cockroachlabs.com)
+
+package security
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/cockroachdb/cockroach/util"
+	"golang.org/x/crypto/bcrypt"
```

Comment:
My understanding is that scrypt is more about key generation than password storage. We don't need to build actual keys, we just prefer to have someone else do the hashing/salting for us.